src/cmd-buildextend-aws

```diff
@@ -15,16 +15,21 @@ from cosalib.cmdlib import run_verbose, write_json, sha256sum_file
 # Parse args and dispatch
 parser = argparse.ArgumentParser()
 parser.add_argument("--build", help="Build ID")
-parser.add_argument("--region", help="EC2 region",
-                    required=True)
-parser.add_argument("--bucket", help="S3 Bucket",
-                    required=True)
+parser.add_argument("--region", help="EC2 region")
+parser.add_argument("--bucket", help="S3 Bucket")
 parser.add_argument("--name-suffix", help="Suffix for name")
 parser.add_argument("--grant-user", help="Grant user launch permission",
                     nargs="*", default=[])
 parser.add_argument("--log-level", help="ore log level")
+parser.add_argument("--upload", action='store_true', default=False,
+                    help="Upload to AWS")
 args = parser.parse_args()
 
+if args.upload:
+    for arg in ['region', 'bucket']:
+        if not args.arg:
```

Comment:
Does this actually work though? I'd expect you to need `getattr` here too.
Could also use that trick in `buildextend-gcp` BTW :)

Reply:
> Does this actually work though? I'd expect you to need `getattr` here too.

oops.. fixed

> Could also use that trick in `buildextend-gcp` BTW :)

I think I'll leave GCP as is because the environment var `GCP_JSON_AUTH` doesn't match the option (`--json-key`).